frontend/app/routes/_gcweb-app.personal-information.address.confirm.tsx

```diff
@@ -96,19 +96,13 @@ export default function ConfirmAddress() {
           </div>
         </div>
 
-        <div className="form-group">
-          <ul className="list-inline lst-spcd">
-            <li>
-              <button id="confirm-button" className="btn btn-primary btn-lg">
-                {t('personal-information:address.confirm.button.confirm')}
-              </button>
-            </li>
-            <li>
-              <Link id="cancel-button" to="/personal-information/address/edit" className="btn btn-default btn-lg">
-                {t('personal-information:address.confirm.button.cancel')}
-              </Link>
-            </li>
-          </ul>
+        <div className="flex flex-wrap gap-3">
+          <button id="confirm-button" className="btn btn-primary btn-lg">
+            {t('personal-information:address.confirm.button.confirm')}
+          </button>
+          <Link id="cancel-button" to="/personal-information/address/edit" className="btn btn-default btn-lg">
+            {t('personal-information:address.confirm.button.cancel')}
+          </Link>
```

Comment:
FYI those `<ul>` were originally taken from WET's style guide: 
https://wet-boew.github.io/wet-boew-styleguide/design/buttons-en.html#btn-groups

I was under the impression we would be using WET/GCWeb's styles wherever we can and Tailwind when we need to do something that the WET/GCWeb styles do not provide.

CC @sebastien-comeau

Reply:
Resolving since this is a non-blocking comment that may or may not need further discussion.

Reply:
Using `ul` to align buttons doesn't make sense, IMO. I do agree we should use WET/GCWeb where it make sense but that template is overkill sometimes and it's based on outdated Bootstrap 3.x.

Reply:
;) it probably will need further discussion.  @sebastien-comeau, what do you want to do?

Reply:
> Using `ul` to align buttons doesn't make sense, IMO. I do agree we should use WET/GCWeb where it make sense but that template is overkill sometimes and it's based on outdated Bootstrap 3.x.

I'm in full agreement with that.

Reply:
> ;) it probably will need further discussion. @sebastien-comeau, what do you want to do?

We can have a general discussion later. With other project like Retirment Hub we used MUI to handle component style and behavior. As for the layout we used Tailwind.